tests/cases/events_test.py

```diff
@@ -136,3 +138,25 @@ def callback(event):
             self.assertEqual(events.daemon.eventQueue.qsize(), 0)
             self.assertEqual(self.ctr, 3)
             self.assertEqual(self.responses, ['foo'])
+            events.daemon.stop()
+
+    @mock.patch.object(events, 'daemon', new=events.ForegroundEventsDaemon())
+    def testForegroundDaemon(self):
```

Comment:
> Am I right to worry about that? Couldn't this test code also work if the standard async daemon is used?

It would create a race condition, and this test would probably fail intermittently if the mock was failing. I added the assertion that the mock was working, there's certainly no harm in that.